demisto_sdk/commands/common/tools.py

```diff
@@ -249,12 +258,13 @@ def get_remote_file(
                 f'Reason: {exc}'
             )
         return {}
+    file_content = res.content if res.ok else local_content
     if return_content:
-        return res.content
+        return file_content
     if full_file_path.endswith('json'):
-        details = res.json()
+        details = res.json() if res.ok else json.loads(local_content)
     elif full_file_path.endswith('yml'):
-        details = yaml.safe_load(res.content)
+        details = yaml.safe_load(file_content)  # type: ignore
```

Comment:
Argument 1 to "safe_load" has incompatible type "object"; expected "Union[bytes, IO[bytes], str, IO[str]]" [arg-type] details = yaml.safe_load(file_content) ^ Found 1 error in 1 file (checked 2 source files)

its weird, since local_content is str and the res.content doesn't give out this error..